src/mono/wasm/build/WasmApp.Common.targets

```diff
@@ -496,9 +496,13 @@
                         '$(_IsToolchainMissing)' == 'true'"
            Text="$(_ToolchainMissingErrorMessage) SDK is required for AOT'ing assemblies." />
 
+    <Error Condition="'$(WasmBuildNative)' == 'false' and '$(WasmSingleFileBundle)' == 'true'"
```

Comment:
this check need to run after `_BoolPropertiesThatTriggerRelinking` caused the change

Reply:
Could you clarify why? `_BoolPropertiesThatTriggerRelinking` will only affect if `$(WasmBuildNative)' == '' `https://github.com/dotnet/runtime/blob/59831acec930dd1213e252501787de09efc23ff5/src/mono/wasm/build/WasmApp.Common.targets#L505 so it shouldn't change the check outcome.

Reply:
oh, it never sets it to false.

Reply:
but if users sets `WasmBuildNative` false and also sets something which would make it true via `_BoolPropertiesThatTriggerRelinking` we have broken combination.